docker-compose.override.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+  core:
+    environment:
+      ANS104_UNBUNDLE_FILTER: '{ "attributes": { "owner_address": "$BUNDLER_ARWEAVE_ADDRESS" }}'
+      ANS104_INDEX_FILTER: '{ "always": true }'
+    profiles: ['bundler']
```

Comment:
when using bundler profile, we will index bundles from our bundler wallet

Reply:
I didn't know about that! profiles are really cool!